src/test/java/org/cdc/gov/sdp/HL7TerserTest.java

```diff
@@ -1,6 +1,7 @@
 package org.cdc.gov.sdp;
```

Comment:
I'm not sure why this isn't highlighting as needing a change but the package name should be gov.cdc....